### DISCUSSION TITLE
F-Droid inclusion

### DISCUSSION THREAD

**Discussion by Poussinou** [CalebQ42/SWAssistant#32] [General]
Hi,

Since this app is MIT licensed, are you considering the fact to add it to [F-Droid](https://f-droid.org/)? There is a strong community behind.

You can find some documentation [here](https://f-droid.org/docs/).

Otherwise, if you don't want to work on the inclusion of this app in F-Droid by yourself, would you still agree with the inclusion of your app in F-Droid by the maintainer team?

**Comment by CalebQ42** [comment]
It's definitely something I want to do, it has just slipped my mind everytime I've meant to do it. Taking a quick look though, I would have to make some changes to make it fully compatible, though it is a fairly small list.
